SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialShadowActor.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+#include "SpatialConstants.h"
```

Comment:
Should include either "CoreMinimal.h" or the headers you need here, e.g. "Actor.h" and (several) others

Reply:
Done